app/views/ministers/index.html.haml

```diff
@@ -1,13 +1,8 @@
 - breadcrumb :ministers
 
-
 %article.content-listing
   %h1
-    Ministers
-  %p.abstract
-    The Australian Government's ministers include Cabinet ministers, outer
-    ministers, assistant ministers, ministers assisting and parliamentary
-    secretaries.
```

Comment:
I was under the impression from @maadonna that all pages had a short_summary. Is this still the case?